build/dependencies.props

```diff
@@ -14,6 +14,8 @@
     <NpgsqlVersion>3.2.0-*</NpgsqlVersion>
     <NpgsqlEntityFrameworkCorePostgreSQLVersion>1.*</NpgsqlEntityFrameworkCorePostgreSQLVersion>
 
+    <RuntimeFrameworkVersion Condition="'$(TargetFramework)'=='netcoreapp2.0'">2.0.0-*</RuntimeFrameworkVersion>
```

Comment:
Oops, wrong answer. This is here to match what we do in other repos. I bet it's there to avoid version mismatches and associated warnings.